assignments/arrays.js

```diff
@@ -63,34 +63,69 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
+console.log(inventory[32]);
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
-
-// ==== Challenge 3 ====
+let lastCar = inventory[inventory.length-1];
+console.log("Make: " + lastCar.car_make + ", Model: " + lastCar.car_model);  
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+
+
+for (let i = 0 ; i < inventory.length ; i++) {
+  
+    let individualInv = inventory[i];
+    
+    // console.log(individualInv);
+    // console.log(individualInv.car_model);
+    carModels.push(individualInv.car_model)
+    console.log(carModels.sort());
+   
+  }
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
 let carYears = [];
-console.log();
+
+for (let i = 0 ; i < inventory.length ; i++) {
+  let invYear = inventory[i]; // individual array item (object)
+  // console.log(invYear);
+  carYears.push(invYear.car_year);
+}
+
+console.log(carYears);
 
 // ==== Challenge 5 ====
 // The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
 let oldCars =[];
-console.log(); 
+
+// iterate through array 
+
+for (let i = 0 ; i < inventory.length ; i++) {
+  // look at car_years
+    // find cars older than the year  2000 => Condition
+  if (inventory[i].car_year < 2000) {
+     // return old cars in oldCar array
+     oldCars.push(inventory[i].car_year); // pushed over and over
+  }
+}
+ console.log(oldCars.length);
 
 // ==== Challenge 6 ====
 // A buyer is interested in seeing only BMW and Audi cars within the inventory.  Return an array that only contains BMW and Audi cars.  Once you have populated the BMWAndAudi array, use JSON.stringify() to show the results of the array in the console.
 let BMWAndAudi =[];
-console.log();
+
+
+for (let i = 0 ; i < inventory.length ; i++) {
+  
+  if (inventory[i].car_make === "BMW" || inventory[i].car_make === "Audi" ) {
+    
+    BMWAndAudi.push(inventory[i].car_make)
+  }
+  
+}
```

Comment:
The only thing you're missing is a line using JSON.stringify() (which turns a JS object/array into a string) and console.log() to print the result.